ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource

```diff
@@ -180,20 +219,25 @@ Delete Data Science Projects From CLI
     END
 
 Delete Data Science Project From CLI
-    [Documentation]    Deletes a single DS project usig CLI command
+    [Documentation]    Deletes a single DS project usig CLI command by project display-name
     [Arguments]     ${displayed_name}
     ${project_k8s_name}=     Run     oc get projects -o json | jq '.items[] | select((.metadata.annotations."openshift.io/display-name" != null) and (.metadata.labels."opendatahub.io/dashboard"=="true") and (.metadata.annotations."openshift.io/display-name"=="${displayed_name}")) | .metadata.name'    # robocop: disable
     ${project_k8s_name}=    Replace String    ${project_k8s_name}    "    ${EMPTY}
     IF    "${project_k8s_name}" == "${EMPTY}"
         Log    msg=There are probably no DS Projects with Display Name equal to ${displayed_name}
         ...    level=WARN
     ELSE
-        Oc Delete    kind=Project   name=${project_k8s_name}
-        WHILE    ${TRUE}
-            ${exists}=    Run And Return Rc    oc get project ${project_k8s_name}
-            IF    "${exists}" == "1"    BREAK
-            Sleep    5s    reason=let's not overload the API
-        END
+        Delete Data Science Project From CLI By Name    name=${project_k8s_name}
+    END
+
+Delete Data Science Project From CLI By Name
+    [Documentation]    Deletes a single DS project usig CLI command by project resource name
+    [Arguments]     ${name}
+    Oc Delete    kind=Project   name=${name}
+    WHILE    ${TRUE}
+        ${exists}=    Run And Return Rc    oc get project ${name}
```

Comment:
instead of the while loop, can't we use something like:
```
oc wait --for=delete --timeout=60s project ${name}
```

maybe we can wait already with the first command to delete the project using `--wait=true`

Reply:
I also don't like that part of the code, but it was already like this in the existing keyword. I'll try to fix it in a follow-up PR